scripts/markdown-link-check.sh

```diff
@@ -35,7 +35,7 @@ ${DOCKER} run ${DOCKER_RUN_OPTS} --rm -t \
   -w /github/workspace \
   --entrypoint /usr/bin/find \
   docker.io/robertbeal/markdown-link-checker \
-  website \( -type f -name "*.md" -or -name "*.markdown" \) -exec markdown-link-check --config .markdownlinkcheck.json --quiet --verbose {} \; \
+  docs \( -type f -name "*.md" -or -name "*.markdown" \) -exec markdown-link-check --config .markdownlinkcheck.json --quiet --verbose {} \; \
```

Comment:
we'll want to update the GHA since it's failing because of the missing files in `docs/d` and `docs/r`